config/kernel-monitor-counter.json

```diff
@@ -0,0 +1,31 @@
+{
+  "plugin": "custom",
+  "pluginConfig": {
+    "invoke_interval": "5m",
+    "timeout": "1m",
+    "max_output_length": 80,
+    "concurrency": 1
+  },
+  "source": "kernel-monitor",
+  "conditions": [
+    {
+      "type": "FrequentUnregisterNetDevice",
+      "reason": "NoFrequentUnregisterNetDevice",
+      "message": "node is functioning properly"
+    }
+  ],
+  "rules": [
+    {
+      "type": "permanent",
+      "condition": "FrequentUnregisterNetDevice",
+      "reason": "UnregisterNetDevice",
+      "path": "/home/kubernetes/bin/log-counter",
```

Comment:
I couldn't get this working without an absolute path.  Any ideas on what the path should be?

Reply:
We can use `https://stackoverflow.com/questions/18537257/how-to-get-the-directory-of-the-currently-running-file`. And document that the path is relative to `node-problem-detector` binary.

It is also fine to enforce absolute path, but we should validate it inside NPD.

Reply:
it didn't seem like it was relative to the npd binary...

Reply:
We do validate the path inside NPD, so it should be alright...

Reply:
> it didn't seem like it was relative to the npd binary...

I mean we can make it so.

Another option is to add a config to config the script root directory. Anyway, I'm fine with starting from this for now.